packages/client/hmi-client/src/page/Home.vue

```diff
@@ -253,9 +396,68 @@ const myFilteredSortedProjects = computed(() => {
 });
 
 const projectsTabs = ref<{ title: string; projects: Project[] }[]>([
-	{ title: 'My projects', projects: [] },
-	{ title: 'Shared projects', projects: [] } // Keep shared projects empty for now
+	{ title: TabTitles.MyProjects, projects: [] },
+	{ title: TabTitles.SharedProjects, projects: [] } // Keep shared projects empty for now
+]);
+
+// Table view
+const columns = ref([
+	{ field: 'name', header: 'Project title' },
+	{ field: 'description', header: 'Description' },
+	{ field: 'username', header: 'Author' },
+	{ field: 'stats', header: 'Stats' },
+	{ field: 'timestamp', header: 'Created' },
+	{ field: 'lastUpdated', header: 'Last updated' } // Last update property doesn't exist yet
```

Comment:
Does columns need to be a ref or can they just be vanilla JS variable?

Reply:
In this case it needs to be - changing the `selectedColumns` stops working as expected